internal/openshift/controller.go

```diff
@@ -70,11 +70,19 @@ func NewController(ctx context.Context, openShiftAPI string, openShiftBearerToke
 // just does couple comparisons and returns.
 func (c *controllerImpl) HandleBuild(o model.Object) error {
 	ns := o.Object.Metadata.Namespace
-	err := c.createIfNotExist(ns)
+	ok, err := c.createIfNotExist(ns)
 	if err != nil {
 		return err
 	}
 
+	if !ok {
```

Comment:
@vpavlin what do you think? This is a temp solution until there would be some labels which could be used to filter events. For now we basically have to assume that an empty tenant info can be ignored.

Reply:
Can't see the code after last commit, so I assume the temp solution is now permament?:-)